interface/forms/care_plan/new.php

```diff
@@ -146,99 +163,111 @@ function set_related(codetype, code, selector, codedesc) {
-                </td>
-                <td align="left" class="forms"><?php echo xlt('Description'); ?>:</td>
-                <td class="forms">
-                    <textarea rows="4" id="description_1" cols="30" name="description[]" class="description"><?php echo text($obj{"description"}); ?></textarea>
-                </td>
-                <td align="left" class="forms"><?php echo xlt('Date'); ?>:</td>
-                <td class="forms">
-                    <input type='text' id="code_date_1" size='10' name='code_date[]' class="code_date datepicker" <?php echo attr($disabled) ?> value='<?php echo attr($obj{"date"}); ?>' title='<?php echo xla('yyyy-mm-dd Date of service'); ?>' />
-                </td>
-                <td align="left" class="forms"><?php echo xlt('Type'); ?>:</td>
-                <td>
-                  <select name="care_plan_type[]" id="care_plan_type_1" class="care_plan_type">
-                    <option value=""></option>
-                    <?php foreach($care_plan_type as $value):
-                      $selected = ($value['value'] == $obj{"care_plan_type"}) ? 'selected="selected"' : '';
-                    ?>
-                    <option value="<?php echo attr($value['value']);?>" <?php echo $selected;?>><?php echo text($value['title']);?></option>
-                    <?php endforeach;?>
-                  </select>
-                </td>
-                <td>
-                    <img src='../../pic/add.png' onclick="duplicateRow(this.parentElement.parentElement);" align='absbottom' width='27' height='24' border='0' style='cursor:pointer;cursor:hand' title='<?php echo xla('Click here to duplicate the row'); ?>'>
-                    <img src='../../pic/remove.png' onclick="deleteRow(this.parentElement.parentElement.id);" align='absbottom' width='24' height='22' border='0' style='cursor:pointer;cursor:hand' title='<?php echo xla('Click here to delete the row'); ?>'>
-                </td>
-            <input type="hidden" name="count[]" id="count_1" class="count" value="1">
-        </tr>
-    <?php }
-    ?>
-
-    <tr>
-        <td align="left" colspan="5" style="padding-bottom:7px;"></td>
-    </tr>
-    <tr>
-        <td colspan="2"></td>
-        <td colspan="3">
-            <input type="hidden" id="clickId" value="">
-            <input type='submit'  value='<?php echo xla('Save'); ?>' class="button-css">&nbsp;
-<!--            <input type='button'  value="Print" onclick="window.print()" class="button-css">-->
-        </td>
-    </tr>
-</table>
-</form>
+		<div class="container">
+        
+			<div class="row">
+				<div class="page-header">
+						<h2><?php echo xlt('Care Plan Form'); ?></h2>
+				</div>
+			</div>
+			<div class="row">
+			<?php echo "<form method='post' name='my_form' " . "action='$rootdir/forms/care_plan/save.php?id=" . attr($formid) . "'>\n"; ?>
```

Comment:
Also, can you break out the `<form>` element into true HTML and only use PHP to echo the needed vars?